backend/telemetry_core/src/state/chain_stats.rs

```diff
@@ -246,22 +251,22 @@ impl ChainStatsCollator {
 
     pub fn generate(&self) -> ChainStats {
         ChainStats {
-            version: self.version.generate_ranking_top(10),
-            target_os: self.target_os.generate_ranking_top(10),
-            target_arch: self.target_arch.generate_ranking_top(10),
-            cpu: self.cpu.generate_ranking_top(10),
+            version: self.version.generate_ranking_top(MAXIMUM_STATS_COUNT),
```

Comment:
Is it possible to add a few comments why 10 vs 1024 is used for certain metrics? 

FYI, I have some troubles to understand the rationale behind by looking at the code, so I would like 
to have either some kind of grouping in by two different structs in ChainStats  or a few comments.

For example:

```
struct Chainstate {
   // This contains full stats for the metrics 
   full_stats: FullStats
   // This contains the top k stats for the metrics 
   top_k_stats: TopStats
}
```



Thoughts?

Reply:
At the moment, staging does not propagate a meaningful number of statistics back to the frontend due to a limited variety of testing nodes with similar kernels and cpus.

I believe we'll end up taking a similar approach to this, but that decision still needs to be data-driven. In the meanwhile, will refactor the `ChainStats` to make it clear for which statistics we'd want more granular control in the frontend, good suggestion! Thanks!